src/MOI_wrapper.jl

```diff
@@ -431,5 +433,79 @@ end
     end
     # qmatind is ::Vector{Cint}, so we convert back to ::Vector{Int}.
     return sparse(Int.(qmatind), qmatcol, qmatval)
- end
+end
+
+function compute_conflict(model::Optimizer)
+    model.conflict = refineconflict(model.inner)
+    return
+end
```

Comment:
Probably just remove this function in favour of explicit calls as outlined in the other comments.